dali/pipeline/operator/sequence_operator.h

```diff
@@ -486,64 +481,55 @@ class SequenceOperator : public Operator<Backend>, protected SampleBroadcasting<
     }
   }
 
-  void InitializeExpandedInputs(const workspace_t<Backend> &ws) {
+  void InitializeExpandedInputs(const Workspace &ws) {
     assert(expanded_.NumInput() == 0);
     for (int input_idx = 0; input_idx < ws.NumInput(); input_idx++) {
       InitializeExpandedInput(ws, input_idx);
     }
     assert(expanded_.NumInput() == ws.NumInput());
   }
 
-  void InitializeExpandedOutputs(const workspace_t<Backend> &ws) {
+  void InitializeExpandedOutputs(const Workspace &ws) {
     assert(expanded_.NumOutput() == 0);
     for (int output_idx = 0; output_idx < ws.NumOutput(); output_idx++) {
       InitializeExpandedOutput(ws, output_idx);
     }
     assert(expanded_.NumOutput() == ws.NumOutput());
   }
 
-  void InitializeExpandedArguments(const workspace_t<Backend> &ws) {
+  void InitializeExpandedArguments(const Workspace &ws) {
     for (const auto &arg_input : ws) {
       auto &shared_tvec = arg_input.second.tvec;
       assert(shared_tvec);
       InitializeExpandedArgument(ws, arg_input.first, *shared_tvec);
     }
   }
 
-  void InitializeExpandedWorkspace(const workspace_t<Backend> &ws) {
-    InitializeExpandedWorkspace(expanded_, ws);
+  void InitializeExpandedWorkspace(const Workspace &ws) {
+    if (ws.HasThreadPool())
+      expanded_.SetThreadPool(&ws.GetThreadPool());
+    expanded_.set_output_order(ws.output_order());
```

Comment:
Is it a snippet put here by mistake? Wanted to point this out just in case.

Reply:
No, it's not a mistake. Previously it happened inside InitializeExpandedWorkspace(workspace_t<XXXBackend> &expanded, const workspace_t<XXXBackend> &ws)`, but these overloads don't work now.